<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolWindowMeasure</class>
 <widget class="QWidget" name="ToolWindowMeasure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Measure</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_main" stretch="0,0,0,1,0,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolBar" name="toolBar">
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <addaction name="actionLine"/>
     <addaction name="actionPolyLine"/>
     <addaction name="actionSpline"/>
     <addaction name="actionRectangle"/>
     <addaction name="actionLabel"/>
     <addaction name="actionContour"/>
     <addaction name="actionDrawOnContour"/>
    </widget>
   </item>
   <item>
    <widget class="Line" name="lineSeparator">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,1,0,0">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="labelId">
       <property name="text">
        <string>Id</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxId"/>
     </item>
     <item>
      <widget class="QLabel" name="labelGroup">
       <property name="text">
        <string>Group</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxGroup"/>
     </item>
     <item>
      <widget class="QtColorPicker" name="colorPickerGroup" native="true">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDeleteDrawing">
       <property name="text">
        <string>Delete Current</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDeleteAllDrawing">
       <property name="text">
        <string>Delete All</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowserInfo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>270</height>
      </size>
     </property>
     <property name="tabStopWidth">
      <number>50</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonLoad">
       <property name="text">
        <string>Load</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSaveAll">
       <property name="text">
        <string>Save All</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonUpdate">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCopy">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Copy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonExport">
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="freeview.qrc">
     <normaloff>:/resource/icons/measure_line.png</normaloff>:/resource/icons/measure_line.png</iconset>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
   <property name="toolTip">
    <string>Line</string>
   </property>
  </action>
  <action name="actionPolyLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="freeview.qrc">
     <normaloff>:/resource/icons/measure_polyline.png</normaloff>:/resource/icons/measure_polyline.png</iconset>
   </property>
   <property name="text">
    <string>PolyLine</string>
   </property>
   <property name="toolTip">
    <string>Polyline</string>
   </property>
  </action>
  <action name="actionSpline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="freeview.qrc">
     <normaloff>:/resource/icons/measure_spline.png</normaloff>:/resource/icons/measure_spline.png</iconset>
   </property>
   <property name="text">
    <string>Spline</string>
   </property>
   <property name="toolTip">
    <string>Spline</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="freeview.qrc">
     <normaloff>:/resource/icons/measure_rectangle.png</normaloff>:/resource/icons/measure_rectangle.png</iconset>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
   <property name="toolTip">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionContour">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="freeview.qrc">
     <normaloff>:/resource/icons/measure_surface_region.png</normaloff>:/resource/icons/measure_surface_region.png</iconset>
   </property>
   <property name="text">
    <string>Contour</string>
   </property>
   <property name="toolTip">
    <string>IsoSurface Region</string>
   </property>
  </action>
  <action name="actionLabel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="freeview.qrc">
     <normaloff>:/resource/icons/measure_label.png</normaloff>:/resource/icons/measure_label.png</iconset>
   </property>
   <property name="text">
    <string>Label</string>
   </property>
   <property name="toolTip">
    <string>Label</string>
   </property>
  </action>
  <action name="actionDrawOnContour">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="freeview.qrc">
     <normaloff>:/resource/icons/measure_surface_region.png</normaloff>:/resource/icons/measure_surface_region.png</iconset>
   </property>
   <property name="text">
    <string>Draw on Contour</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtColorPicker</class>
   <extends>QWidget</extends>
   <header>qtcolorpicker.h</header>
   <container>1</container>
   <slots>
    <signal>colorChanged(QColor)</signal>
    <signal>currentColorChanged(QColor)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="freeview.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>spinBoxId</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ToolWindowMeasure</receiver>
   <slot>OnSpinBoxId(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxGroup</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ToolWindowMeasure</receiver>
   <slot>OnSpinBoxGroup(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonLoad</sender>
   <signal>clicked()</signal>
   <receiver>ToolWindowMeasure</receiver>
   <slot>OnLoad()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSave</sender>
   <signal>clicked()</signal>
   <receiver>ToolWindowMeasure</receiver>
   <slot>OnSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>354</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSaveAll</sender>
   <signal>clicked()</signal>
   <receiver>ToolWindowMeasure</receiver>
   <slot>OnSaveAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonUpdate</sender>
   <signal>clicked()</signal>
   <receiver>ToolWindowMeasure</receiver>
   <slot>OnUpdate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCopy</sender>
   <signal>clicked()</signal>
   <receiver>ToolWindowMeasure</receiver>
   <slot>OnCopy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonExport</sender>
   <signal>clicked()</signal>
   <receiver>ToolWindowMeasure</receiver>
   <slot>OnExport()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorPickerGroup</sender>
   <signal>colorChanged(QColor)</signal>
   <receiver>ToolWindowMeasure</receiver>
   <slot>OnColorGroup(QColor)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonDeleteDrawing</sender>
   <signal>clicked()</signal>
   <receiver>ToolWindowMeasure</receiver>
   <slot>OnButtonDelete3DRegion()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>560</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonDeleteAllDrawing</sender>
   <signal>clicked()</signal>
   <receiver>ToolWindowMeasure</receiver>
   <slot>OnButtonDeleteAll3DRegions()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>608</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnSpinBoxId(int)</slot>
  <slot>OnSpinBoxGroup(int)</slot>
  <slot>OnLoad()</slot>
  <slot>OnSave()</slot>
  <slot>OnSaveAll()</slot>
  <slot>OnUpdate()</slot>
  <slot>OnCopy()</slot>
  <slot>OnExport()</slot>
  <slot>OnColorGroup(QColor)</slot>
  <slot>OnButtonDelete3DRegion()</slot>
  <slot>OnButtonDeleteAll3DRegions()</slot>
 </slots>
</ui>
